### PR TITLE
fix deploy and check scripts; enable disabled tests

### PR DESCRIPTION
**Pull request by DavidMinarsch**
## Proposed changes

This PR applies a few minor fixes which are possible post-release.

## Fixes

-

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have checked that the documentation about the `aea cli` tool works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

-

### CODE REVIEW

File: scripts/deploy_to_registry.py [hunk 1]
```diff
@@ -40,6 +40,8 @@
 DEFAULT_CONFIG_FILE_PATHS = [
     Path("aea", "connections", "stub", "connection.yaml"),
     Path("aea", "protocols", "default", "protocol.yaml"),
+    Path("aea", "protocols", "signing", "protocol.yaml"),
+    Path("aea", "protocols", "state_update", "protocol.yaml"),
     Path("aea", "skills", "error", "skill.yaml"),
```
Comment (marcofavorito) [inline scripts/deploy_to_registry.py:45]: what about `ledger` connection and `ledger_api`/`contract_api` protocols?

Comment (marcofavorito) [reply scripts/deploy_to_registry.py:45]: maybe in a separate PR

Comment (DavidMinarsch) [reply scripts/deploy_to_registry.py:45]: they are not in the `aea` package so not relevant.

Comment (marcofavorito) [reply scripts/deploy_to_registry.py:45]: should they, though?

**Pull request merged**